Makefile

```diff
@@ -99,6 +100,26 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg repair
 	stg import -s ../patch/series
 
+	mkdir $(NON_UP_LOC)
+
+	if [ ! -z ${EXTERNAL_KERNEL_PATCHES} ];  then
+		wget ${EXTERNAL_KERNEL_PATCHES} -O patches.tar.gz
+		tar -zxf patches.tar.gz -C $(NON_UP_LOC)
+	fi
+
+	# Precedence is given for external URL
+	if [ -z ${EXTERNAL_KERNEL_PATCHES} ] && [ x${INCLUDE_MLNX_PATCHES} == xy ]; then
+		if [ -f "../../../$(MLNX_PATCH_LOC)" ]; then
```

Comment:
Can MLNX_PATCH_LOC be an absolute path ?

Reply:
Updated, will be set in `platform/mellanox/rules.mk`